api/lib/domain/constants.js

```diff
@@ -91,6 +91,13 @@ module.exports = {
     },
   },
 
+  OIDC_ERRORS: {
+    USER_INFO: {
+      missingFields: { shortCode: 'OIDC01', code: 'USER_INFO_MANDATORY_MISSING_FIELDS' },
```

Comment:
Il serait préférable que la propriété `USER_INFO.missingFields` et la constante `USER_INFO_MISSING_FIELDS` soient cohérentes et si une erreur est renvoyée, c'est que c'était « mandatory » 😉 :

```suggestion
      missingFields: { shortCode: 'OIDC01', code: 'USER_INFO_MISSING_FIELDS' },
```

Reply:
Done